build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml

```diff
@@ -84,6 +84,49 @@ jobs:
       artifactName: 'mrtcore_test_binaries_$(buildPlatform)'
       downloadPath: $(testPayloadDir)
 
+  - script: |
+      echo source: $(Build.SourcesDirectory)\BuildOutput\Release\$(buildPlatform)
+      echo dest: $(Agent.TempDirectory)\pushnotification_testbin\$(buildPlatform)
+      dir $(Build.SourcesDirectory)\BuildOutput\Release\$(buildPlatform)
+      dir D:\
+      dir D:\a\
+      dir D:\a\1\
+      dir D:\a\1\s\
+      dir D:\a\1\s\BuildOutput\
+      dir D:\a\1\s\BuildOutput\Release\
+      dir D:\a\1\s\BuildOutput\Release\x64
+      dir D:\a\1\s\BuildOutput\Release\x64\PushNotificationTests
+    displayName: 'Copy push notification tests debug'
```

Comment:
Where's the 'copy'?

P.S. If this is debugging-printfery you may want to consider fewer/one `DIR /S`

Reply:
Yeah, that was debugging and I didn't think of using /s. 

So it looks like the entire solution is being built and I can see that the pushnotificationtests project is built and I would expect the output to be in $(Build.SourcesDirectory)\BuildOutput\Release\$(buildPlatform) but I didn't. The only thing in that directory is the helix test directory.

I'm sure that the buildoutput for pushtnotificationtests existed at some point, but by the time I get to executing the helix template, it's gone. The question is where /when does it get wiped out? I couold grab a copy before then and then use that copy to publish it for helix.

Reply:
The copy is just after that chunck of debuging printfery code. I was expecting to be able to copy the pushnotifitcationtests folder under BuildOutput\Release\x64 or x86 depending but the pushnotifitcationtests folder doesn't seem to exist anymore at this point in the build process.